pkgs/by-name/os/ossia-score/package.nix

```diff
@@ -0,0 +1,144 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, ninja
+, pkg-config
+, alsa-lib
+, avahi
+, bluez
+, boost185
+# , faust
```

Comment:
Is this supposed to be commented out? If so, could you leave a small comment about why it is here?

Reply:
hmm this was in the previous PR i sourced the package.nix from. This is an optional feature which apparently doesn't work yet in nix.